### PR TITLE
feat(COR-1674): Added the new lokalize key

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard30**
- Added the new lokalize key _population_count_connected_to_rwzis_

### CODE REVIEW

File: packages/cms/src/studio/data/data-structure.ts
Comment (VWSCoronaDashboard18) [inline packages/cms/src/studio/data/data-structure.ts:1]: This file should not be part of this PR, I think...

**Review by VWSCoronaDashboard18 — CHANGES_REQUESTED** [review]
See my comment, the other two files are great!

**Review by VWSCoronaDashboard28 — APPROVED** [review]

**Review by VWSCoronaDashboard18 — CHANGES_REQUESTED** [review]
test for realtime github updates

**Review by VWSCoronaDashboard18 — APPROVED** [review]
Now it's OK to merge. Because the other PR's (where the data structure changes) are on the way.
When the data structure is deleted in this PR, a new commit needs to happen, and two new approvals are needed. So for time/effort constraints we do it already in this PR

**Pull request merged**